### PR TITLE
ui/15519199-colony-widget

### PR DESCRIPTION
**Pull request by krawczykkatarzyna**
Screen Shot 2023-07-17 at 09 59 36
Screen Shot 2023-07-17 at 09 59 41

### CODE REVIEW

File: src/components/v5/shared/ColonyVersionWidget/ColonyVersionWidget.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        'flex rounded border border-gray-200 bg-base-white px-5 py-[1.125rem]',
+        {
+          'flex-col': isMobile,
+        },
+      )}
+    >
+      <div
+        className={clsx('flex items-center gap-6 w-full', {
+          'flex-col mb-6': isMobile,
+          'flex-row mr-6': !isMobile,
```
Comment (CzarekDryl) [inline src/components/v5/shared/ColonyVersionWidget/ColonyVersionWidget.tsx:33]: Is flew-row needed?

File: src/components/v5/shared/ColonyVersionWidget/ColonyVersionWidget.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+          </div>
+        </div>
+
+        {isMobile ? (
+          <div className="w-full bg-gray-200 h-px" />
+        ) : (
+          <div className="bg-gray-200 h-full w-px" />
+        )}
+
+        <div className={`${styles.wrapper} flex flex-col justify-center`}>
```
Comment (CzarekDryl) [inline src/components/v5/shared/ColonyVersionWidget/ColonyVersionWidget.tsx:60]: Please move all styles to `styles.wrapper`

File: src/components/v5/shared/ColonyVersionWidget/ColonyVersionWidget.tsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+            >
+              <Icon
+                appearance={{ size: 'extraTiny' }}
+                name={status === 'success' ? 'check-circle' : 'warning-circle'}
+              />
+            </span>
+            <span className={styles.value}>{currentVersion}</span>
+          </div>
+        </div>
+
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/v5/shared/ColonyVersionWidget/ColonyVersionWidget.tsx:53]: `<div className={clsx('bg-gray-200', { 'w-full h-px': isMobile, 'h-full w-px': !isMobile })}/>`

File: src/components/v5/shared/ColonyVersionWidget/ColonyVersionWidget.module.css [hunk 4]
```diff
@@ -0,0 +1,11 @@
+.text {
+  @apply flex gap-1 items-center justify-center;
+}
+
+.value {
+  @apply text-gray-900 heading-5;
```
Comment (CzarekDryl) [inline src/components/v5/shared/ColonyVersionWidget/ColonyVersionWidget.module.css:6]: Is text-gray-900 needed?

**Review by KatarzynaStefanowiczPagepro — APPROVED** [review]

**Pull request merged**